packages/airnode-examples/README.md

```diff
@@ -162,26 +162,37 @@ Refer to the
 [documentation](https://docs.api3.org/airnode/latest/grp-providers/guides/build-an-airnode/configuring-airnode.html) for
 more details.
 
-### 9. Build docker artifacts
+### 9. A Note on Docker Images
+
+API3 makes docker container builds for releases available on
+[Docker Hub](https://hub.docker.com/u/api3/airnode-deployer). API3 also makes development builds available; if this
```

Comment:
I suggest removing `airnode-deployer` from this link since it's meant to point to more than just the deployer build.
```suggestion
[Docker Hub](https://hub.docker.com/u/api3/). API3 also makes development builds available; if this
```